packages/flutter/test/material/refresh_indicator_test.dart

```diff
@@ -455,4 +455,71 @@ void main() {
 
     expect(layoutCount, 1);
   });
-}
\ No newline at end of file
+
+  testWidgets('strokeWidth cannot be null in RefreshIndicator', (WidgetTester tester) async {
+    try {
+      await tester.pumpWidget(
+          MaterialApp(
+            home: RefreshIndicator(
+              onRefresh: () async {},
+              strokeWidth: null,
+              child: ListView(
+                physics: const AlwaysScrollableScrollPhysics(),
+                children: <String>['A', 'B', 'C', 'D', 'E', 'F'].map<Widget>((String item) {
+                  return SizedBox(
+                    height: 200.0,
+                    child: Text(item),
+                  );
+                }).toList(),
+              ),
+            ),
+          )
+      );
+    } on AssertionError catch(_){
+      return;
+    }
+    fail('The assertion was not thrown when strokeWidth was null');
+  });
+
+  testWidgets('RefreshIndicator responds to strokeWidth', (WidgetTester tester) async {
+    await tester.pumpWidget(
+        MaterialApp(
+          home: RefreshIndicator(
+            onRefresh: () async {},
+            child: ListView(
+              physics: const AlwaysScrollableScrollPhysics(),
+              children: <String>['A', 'B', 'C', 'D', 'E', 'F'].map<Widget>((String item) {
+                return SizedBox(
+                  height: 200.0,
+                  child: Text(item),
+                );
+              }).toList(),
+            ),
+          ),
+        )
+    );
+
+    //By default the value of strokeWidth is 2.0
+    expect(tester.widget<RefreshIndicator>(find.byType(RefreshIndicator)).strokeWidth,2.0);
```

Comment:
Per the style guide we try to keep line length to 80 characters when feasible. :)

```suggestion
    expect(
      tester.widget<RefreshIndicator>(find.byType(RefreshIndicator)).strokeWidth,
      2.0,
    );
```

Reply:
Oh, sorry I was trying to occupy less verticle space. I have followed the 80 characters line length in the recent commit.